### PR TITLE
ssl_openssl_lib.pas: Fixed copy-paste mistake.

### PR DESCRIPTION
**Pull request by k0ste**
@PeterDaveHello note about this [here](https://github.com/leonsoft-kras/transmisson-remote-gui/pull/966/commits/9bba34167746ecc0108071f0a5d29dc62ebf1f66#r125749430).

### CODE REVIEW

File: synapse/source/lib/ssl_openssl_lib.pas [hunk 1]
```diff
@@ -1867,7 +1867,7 @@ function InitSSLInterface: Boolean;
       Result := TRUE;
       exit;
     end;
-  {/pf}  
+  {/pf}
```
Comment (PeterDaveHello) [inline synapse/source/lib/ssl_openssl_lib.pas]: Is this change related?

Comment (k0ste) [inline synapse/source/lib/ssl_openssl_lib.pas]: This changes made by editor.

Comment (PeterDaveHello) [inline synapse/source/lib/ssl_openssl_lib.pas]: Personally I prefer not to add anything unrelated to make the code and commit clean, mind to remove this change?

Comment (k0ste) [inline synapse/source/lib/ssl_openssl_lib.pas]: This is parasite whitespaces. They should not exist at all.

Comment (PeterDaveHello) [inline synapse/source/lib/ssl_openssl_lib.pas]: Then we better use a separated commit to fix that.

Comment (k0ste) [inline synapse/source/lib/ssl_openssl_lib.pas]: As you wish. Added two parasite whitespaces.
